src/components/UI/FileUpload/FileUpload.tsx

```
@@ -0,0 +1,68 @@
+import styles from './FileUpload.module.css';
+import CrossIcon from 'assets/images/icons/Cross.svg?react';
+import UploadIcon from 'assets/images/icons/UploadIcon.svg?react';
+import { slicedString } from 'common/utils';
+import { useState } from 'react';
+
+interface FileUploadProps {
+  setErrors: (errors: { message: string }[]) => void;
+  afterImport: (content: String | null | ArrayBuffer, name: string) => void;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)